src/utilities.cpp

```diff
@@ -157,10 +157,6 @@ std::string ToString(Precision value) {
 // Helper for the below function to convert the argument to the value type. Adds specialization for
 // complex data-types. Note that complex arguments are accepted as regular values and are copied to
 // both the real and imaginary parts.
-template <typename T>
-T ConvertArgument(const char* value) {
-  return static_cast<T>(std::stoi(value));
-}
```

Comment:
Why move this to the header? Because of the template otherwise not being instantiated? That makes sense, but the rest of CLBlast contains only declarations in the headers and the implementation + optional explicit instantiation in the .cpp file. Perhaps it is nicer to keep it here and than explicitly instantiate it.